### ISSUE TITLE
A question about the pre-trained model

### ISSUE THREAD

**Issue by Honlan** [vt-vl-lab/Guided-pix2pix#4]
Hi, thanks a lot for your awesome work.

I downloaded the data of DeepFashion in pickle format, trained the model with the following script as you suggest.

`python train.py --dataroot /root/DeepFashion/ --name exp_name --netG bFT_resnet --dataset_mode pose --input_nc 3 --guide_nc 18 --output_nc 3 --lr 0.0002 --niter 100 --niter_decay 0 --batch_size 8 --use_GAN --netD basic --beta1 0.9 --checkpoints_dir ./pose_checkpoints`

However, the test results of the trained model don't agree with those generated by your pre-trained model. 

the result generated by my model
![8](https://user-images.githubusercontent.com/5653648/74138446-975dcb80-4c2c-11ea-897f-737466112841.png)

the result generated by your pre-trained model
![8](https://user-images.githubusercontent.com/5653648/74138539-beb49880-4c2c-11ea-871e-60cd1ad37259.png)

The pre-trained model you provide seems to generate results that are more clear and sharp.

Could you please give me some suggestions? Thanks.

**Comment by Honlan** [comment]
More examples.

My model
![9](https://user-images.githubusercontent.com/5653648/74138876-51553780-4c2d-11ea-8a58-e14174d444bc.png)

Your pre-trained model
![9](https://user-images.githubusercontent.com/5653648/74138890-587c4580-4c2d-11ea-9d9f-f26ae0d3e456.png)

**Comment by BadourAlBahar** [comment]
Please note that for our final results we train for 100 epochs using a learning rate of
0.0002 with a batch size of 8, then we minimize the learning rate to 0.00002 and train for 50 additional epochs.

**Comment by Honlan** [comment]
> Please note that for our final results we train for 100 epochs using a learning rate of
> 0.0002 with a batch size of 8, then we minimize the learning rate to 0.00002 and train for 50 additional epochs.

Thanks a lot for you reply. So you train 100 epochs with lr=0.0002, then 50 epochs with lr=0.00002 ?

Another question. I set the batch_size as 32. Will the batch_size influence the final results?

**Comment by BadourAlBahar** [comment]
Yes. 
The results for the first 100 epochs with lr=0.0002 are shown in the ablation study both qualitative and quantitative. 
The final results after continuing training for an additional 50 epochs with lr=0.00002 are shown in the results comparison w/ state-of-the-art. 
As for the batch size. I am not sure. You can try it out and see.

**Comment by Honlan** [comment]
Got it. Thanks a lot.
